features/cloud.py

```diff
@@ -151,19 +151,8 @@ def launch(
             inbound_ports=inbound_ports,
         )
         logging.info(
-            "--- {} instance launched: {}. Waiting for ssh access".format(
-                self.name, inst.name
-            )
+            "--- {} instance launched: {}".format(self.name, inst.name)
         )
-        time.sleep(15)
-        for sleep in (5, 10, 15):
```

Comment:
Why are we removing this retry mechanism ?

Reply:
From conversation: I'll make a separate PR for this and we'll see if it breaks anything :upside_down_face:

Reply:
https://github.com/canonical/ubuntu-advantage-client/pull/2347